bagit.py

```diff
@@ -137,7 +138,8 @@ def find_locale_dir():
 
 
 def make_bag(
-    bag_dir, bag_info=None, processes=1, checksums=None, checksum=None, encoding="utf-8"
+    bag_dir, bag_info=None, processes=1, checksums=None, checksum=None,
+    encoding="utf-8", line_length=0
```

Comment:
I'm wondering whether `line_length` should be something like `max_tag_line_length` so it's a little more obvious what it applies to

Reply:
I agree `line_length` is a bit too general. In thinking what would make sense with an option named `--wrap-tags` (c.f. conversation above), I worry `max_tag_line_length` might be too much:

```
  --wrap-tags MAX_TAG_LINE_LENGTH
                        Limit line lengths in the tag file (bag-info.txt) by
                        wrapping long values and indenting subsequent lines
                        with a space character. (Default: don't.)
```

What about `tag_wrap_column`? That leads to:

```
  --wrap-tags TAG_WRAP_COLUMN
                        Limit line lengths in the tag file (bag-info.txt) by
                        wrapping long values and indenting subsequent lines
                        with a space character. (Default: don't.)
```